pkg/storage/storage_test.go

```diff
@@ -287,7 +288,7 @@ var _ = Describe("In-memory storage", func() {
 
 	It("should return nil metrics when a pod was added in the last scrape", func() {
 		newPodPoint := PodMetricsPoint{Name: "pod2", Namespace: "ns2", Containers: []ContainerMetricsPoint{
-			{Name: "container1", MetricsPoint: newMetricsPoint(now.Add(900*time.Millisecond), 910, 920)},
+			{Name: "container1", MetricsPoint: newMetricsPoint(time.Now(), now.Add(900*time.Millisecond), 910, 920)},
```

Comment:
It might be better to use the `now` variable here to prevent any flakes. It is very unlikely that a slight change in start time will make this test fail, but it is better to have a fixed value.